src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
-</html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link 
+      rel="preconnect" 
+      href="https://fonts.gstatic.com" 
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <img 
+        src="./images/imac.jpeg" 
+        alt="Picture of a computer" 
+        class="card__img card__img--indent"
+      >
+
+      <p class="card__title1 card__title1--indent">Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)</p>
+      <p class="card__title2 card__title2--indent">Код товара: 195434</p>
+
+      <div class="star star--indent">
+        <div class="star__list">
```

Comment:
```suggestion
        <div class="star__list star__list--4">
```
Тут ще повинен бути клас-модифікатор, який визначає скільки саме активних зірочок має бути